### ISSUE TITLE
[solution-checker] Fail job on Medium severity issue

### ISSUE THREAD

**Issue by garrytrinder** [microsoft/powerplatform-actions#125]
Not sure if this is documented anywhere but I'd like to be able to fail a job if the `solution-checker` returns medium severity issues.

When building Canvas Apps it is very easy to overlook the accessibility checker warnings which are raised as Medium severity warnings. I'd like to be able to ensure that we have quality gates in place so that developers cannot overlook this so failing the job will help enforce this.

![image](https://user-images.githubusercontent.com/11563347/135233781-9f666b1f-a15a-445a-95ac-f14b7f0f88b9.png)

**Comment by jjkinsey** [comment]
Seconding this! It would be great to e.g. have access to the counts of each error type as outputs of the action--that way we can create whatever checks are necessary for a given use of the check-solution action (maybe different standards for releasing to testing vs. production).

**Comment by javiercheke** [comment]
Hi, I saw this in the Power Platform Tools on Azure Devops but Can I run the solution checker with Github Actions? This actions is available?

**Comment by echvsg** [comment]
@garrytrinder  or @jjkinsey : Were you using service principal to run this check-solution action. For some reason it says the service principal is disabled but i can use the same service principal without an issue from pac cli check solution command. 
 
 Error: AADSTS500014: The service principal for resource 'https://service.powerapps.com/' is disabled. This indicate that a subscription within the tenant has lapsed, or that the administrator for this tenant has disabled the application, preventing tokens from being issued for it.

Any thoughts?
